src/lib/blueprint.ts

```diff
@@ -1,22 +1,36 @@
 import type { Openapi } from './openapi.js'
 
-export interface Blueprint {
+export interface Blueprint<T extends TypesModule> {
   name: string
-  routes: Route[]
+  routes: Array<Route<T>>
+  resources: Partial<Record<ResourceType<T>, Resource<T>>>
```

Comment:
Added the Partial type to avoid type error. Potentially will be deleted in the future